articles/azure/azs-how-resize-disk-ps.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to resize a disk using PowerShell | UKCloud Ltd
+description: Provides help for resizing a disk using PowerShell on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+reviewer: BaileyLawson
+lastreviewed: 14/03/2019 17:00:00
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Resize a disk
+toc_sub3:
+toc_sub4:
+toc_title: Resize a disk - PowerShell
+toc_fullpath: Users/How To/Resize a disk/azs-how-resize-disk-ps.md
+toc_mdlink: azs-how-resize-disk-ps.md
+---
+
+# How to resize a disk using PowerShell
```

Comment:
Similar comment as metadata - make consistent between this article and the portal article

Reply:
Fixed here: https://github.com/UKCloud/documentation/pull/542